1.9/usage/debugging/cli-debugging/task-exec/index.md

```diff
@@ -30,4 +30,4 @@ You can execute this command in any of the following four modes.
 For more information, see:
 
 - [Command reference](/docs/1.9/usage/cli/command-reference/)
```

Comment:
Bot of these bullets should have periods, or neither. Also, both should be sentence case, or neither.